lib/storage-service.js

```diff
@@ -145,7 +148,7 @@ class StorageService {
     const manager = new LeaseManager(provider, secretID);
 
     StorageService.setEvents(manager);
-    StorageService.logManagerEvent('DEBUG', 'Created LeaseManager.', manager);
+    StorageService.logManagerEvent('INFO', 'Created LeaseManager.', manager);
```

Comment:
Do we want to update the [StorageService#logManagerEvent](https://github.com/rapid7/tokend/blob/master/lib/storage-service.js#L158-L174) so it logs the correlation ID?